src/smart-components/workspaces/overview/about-access-tab.tsx

```diff
@@ -0,0 +1,254 @@
+        icon={<img src={workspacesIcon} alt="workspaces-header-icon" />}
+        linkProps={{
+          label: 'Learn more',
+          isExternal: true,
+          // to do - add learn more url once available
+        }}
+      />
+      <PageSection>
+        <Title headingLevel="h2" className="pf-u-mb-md" data-ouia-component-id="header-title">
+          Get started with workspaces
+        </Title>
+        <Text component={TextVariants.p}>
+          Workspaces let&apos;s you group related assets together (such as RHEL hosts). This simplifies management and user access control.
+        </Text>
+        <br></br>
+        <ExpandableSection
+          toggleText="Show me how my assets and permissions will be organized into workspaces"
+          onToggle={onToggle}
+          isExpanded={isExpanded}
+        >
+          {/* to do - add migration visualization when ready */}
+          <Card ouiaId="animation-card">
+            <CardTitle>A cool animation</CardTitle>
+            <CardBody>will go here</CardBody>
+            <CardFooter>when its ready</CardFooter>
+          </Card>
+        </ExpandableSection>
+
+        <br></br>
+
+        <Gallery hasGutter minWidths={{ default: '330px' }}>
+          <GalleryItem>
+            <ServiceCard
+              isFullHeight
+              title="Workspaces"
+              subtitle=""
+              description="Configure workspaces to fit your organizational structure. They can be structured in a heirarchy (parent-child relationships).
+                Permissions assigned to a parent workspace are automatically inherited by its child workspaces, saving you congfiguration time."
+              icon={<InfrastructureIcon className="pf-u-primary-color-100 pf-v5-c-icon pf-m-lg" />}
+              footer={
+                <Button variant={ButtonVariant.primary} isInline component="a" href="">
+                  Workspaces
+                </Button>
+              }
+              ouiaId="workspaces-service-card"
+            />
+          </GalleryItem>
+          <GalleryItem>
+            <ServiceCard
+              isFullHeight
+              title="Groups"
+              subtitle=""
+              description="Create user groups of both end-users and service accounts. Tailor these groups to mirror your organization's structure."
+              icon={<UsersIcon className="pf-u-primary-color-100 pf-v5-c-icon pf-m-lg" />}
+              footer={
+                <Button variant={ButtonVariant.secondary} isInline component="a" href="/iam/user-access/groups">
+                  View groups
+                </Button>
+              }
+              ouiaId="groups-service-card"
+            />
+          </GalleryItem>
+          <GalleryItem>
+            <ServiceCard
+              isFullHeight
+              title="Role"
+              subtitle=""
```

Comment:
```suggestion
```

Reply:
it errors when I try and remove subtitle, that's why I added it as an empty string
`Property 'subtitle' is missing in type '{ isFullHeight: true; title: string; description: string; icon: Element; footer: Element; ouiaId: string; }' but required in type 'ServiceCardProps'.ts(2741)`

Reply:
Oh yeah subtitle isn't an optional prop. I could open up a PR to make it so if you want?

Reply:
Could be worth it? I know this is all for this specific use case, but if UX is designing in this way, might make sense.

Reply:
went ahead and made a pr https://github.com/patternfly/react-component-groups/pull/261